interchain-token-service/src/events.rs

```diff
@@ -0,0 +1,97 @@
+use cosmwasm_std::{Attribute, Event};
+use router_api::{Address, ChainName, CrossChainId};
+
+use crate::primitives::ItsMessage;
+
+pub enum ItsContractEvent {
```

Comment:
you could just name this `Event`, it's sort of redundant to call it `ItsContractEvent` since the containing package gives you the scoping